ui/component/fileViewCountInline/view.jsx

```diff
@@ -30,13 +30,16 @@ export default function FileViewCountInline(props: Props) {
   // clean up (only one place edit/remove).
   const isChannelPage = window.location.pathname.startsWith('/@');
 
+  // dont show if no view count, if it's a repost, a livestream or isn't a channel page
   if (!viewCount || (claim && claim.repost_url) || isLivestream || !isChannelPage) {
```

Comment:
Feels like the variables names are already self-explanatory?  There's no additional info here.